### PR TITLE
Accessabiliy issue order of headings

### PR DESCRIPTION
**Pull request by brittainmark**
Need to change itemTitle from `<h5>` to `<p>` to score 100 on accessability in google chrome

### CODE REVIEW

File: docs/bootstrap/readme.html [hunk 1]
```diff
@@ -194,11 +194,12 @@ <h3>Version-Specific Upgrade Suggestions</h3>
     <h2>Version History</h2>
     <p>This section identifies the template's version history and changes, starting with its v2.0.0c release.  Prior version changes can be found in the root of the template's distribution zip-file (<code>CHANGELOG.txt</code>).</p>
     <ul>
-        <li>v3.6.5-beta2, 2024-04-02 (lat9)<ul>
+        <li>v3.6.5-beta3, 2024-04-03 (brittainmark)<ul>
```
Comment (lat9) [inline docs/bootstrap/readme.html:197]: This should be `lat9, brittainmark`.

File: docs/bootstrap/readme.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
 
 <body>
     <h1>Bootstrap-4 Template for Zen Cart v1.5.7, v1.5.8 and v2.0.0</h1>
-    <p>Version 3.6.5, by lat9, drbyte, scottcwilson, marco-pm, dbltoe, dennisns7d, brittainmark, torvista, proseLA and daphilli224</p>
+    <p>Version 3.6.5, by lat9, drbyte, scottcwilson, marco-pm, dbltoe, dennisns7d, brittainmark, torvista, proseLA, daphilli224 and brittainmark</p>
```
Comment (lat9) [inline docs/bootstrap/readme.html:72]: You're already in the list; restore to the previous version.

**Review by lat9 — CHANGES_REQUESTED** [review]
Thanks for taking care of all of the doc updates!  Just a couple of minor changes.

**Comment by brittainmark** [comment]
I think there are a number of these, do you want them one at a time or in one lot?

**Review by lat9 — APPROVED** [review]

**Comment by lat9** [comment]
This is associated with issue #352

**Pull request merged**